ultradns-rest/src/main/resources/application.properties

```diff
@@ -0,0 +1,7 @@
+# Feign Client configuration
```

Comment:
If we add this to this file i.e. application.properties other than classpath, then we have to explicitly mention the classpath while invoking denominator jar. So it will have the dependency to client which we don't want.